docs/source/technical_design/containers.md

```diff
@@ -38,7 +38,7 @@ Make sure that you also add the keys and values from `.env-defaults` to your `.e
 Now you can pull and start the containers using the following command:
 
 ```shell
-docker-compose -f docker-compose.release-example.yml up -d
+docker compose --env-file .env-prod -f docker-compose.release-example.yml up -d
```

Comment:
The docker-compose.release-example.yml already uses .env-prod directly, so I don't think we need to specify this again?

Reply:
Unfortunately, we do. `--env-file` must be specified explicitly through the CLI for the `:${KAT_VERSION}` tag to work. If not explicitly provided, `docker-compose` will only load the shell environment and `.env`.